File: golang/http-modify-header/handler.go
package main

import (
	"fmt"
	"net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
	// Get the request headers
	headers := r.Header

	// Modify the headers
	headers.Set("User-Agent", "Boltic Serverless")

	// Print the modified headers (optional)
	fmt.Println("Modified Headers:", headers)

	// Write a response with modified headers
	w.WriteHeader(http.StatusOK)
}

File: nodejs/http-modify-header/handler.js
// Define the handler function
export const handler = async (req, res) => {
  // Get the request headers
  const headers = req.headers;

  // Modify the headers
  headers["user-agent"] = "Boltic Serverless";

  // Return a response with modified headers
  res.status(200).send({ message: "Modified Headers", headers });
};
